htsohm/material_files.py

```diff
@@ -439,7 +439,7 @@ def write_pseudo_atoms(psu_file, atom_types):
     with open(psu_file, "w") as file:
         file.write(
             "#number of pseudo atoms\n" +
-            "%s\n" % (len(atom_types) + 8) +
+            "%s\n" % (len(atom_types) + 10) +
```

Comment:
This is because we are adding Xe / Kr to the raspa file, even if we aren't using them?